Docs: Flow control section (#754)
inngest/website: pages/docs/reference/functions/run-priority.mdx

# Function run priority <VersionBadge version="v3.2.1+" />

> Defining a function's run priority allows you to **prioritize specific function runs above other runs within the same function**.

Reviewer: ```suggestion
You can prioritize specific function runs above other runs **within the same function**.
```
